packages/odata-v4/src/entity-base.ts

```diff
@@ -0,0 +1,5 @@
+export {
```

Comment:
Do we need this?

Reply:
Good catch, I forgot to delete it after our new approach